dev-ops/runAnsible.sh

```diff
@@ -0,0 +1,3 @@
+echo "Mike and Rob's devboxes sometimes throw errors so I've taken to excluding them from the hosts file when I get impatient. - Teo Nov 7, 2016"
```

Comment:
Is this a hint that the user might want to do this? If so, how? Currently I see the dev boxes in the hosts file